packages/autocannon-compare-cli/test/util.test.js

```diff
@@ -8,38 +10,38 @@ import {
   Kind
 } from '../util.js'
 
-tap.test('formatBytes', async t => {
-  t.equal(formatBytes(100), '100 bytes')
-  t.equal(formatBytes(1024), '1.0 KB')
-  t.equal(formatBytes(1024000), '1000 KB')
-  t.equal(formatBytes(1073741824), '1.0 GB')
+test('formatBytes', async () => {
+  assert.equal(formatBytes(100), '100 bytes')
```

Comment:
.equal is depecated in favor of strictEqual https://nodejs.org/api/assert.html#assertequalactual-expected-message

Reply:
Solved